packages/react-components/react-theme-sass/jest.config.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
+
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
+module.exports = {
+  displayName: 'react-theme-sass',
+  preset: '../../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/tsconfig.spec.json',
+      diagnostics: false,
+    },
+  },
+  testEnvironment: 'jest-environment-node-single-context',
```

Comment:
Required for `dart-sass` used in Jest: https://github.com/sass/dart-sass/issues/1692